packages/frontend/src/components/ProjectConnection/index.tsx

```diff
@@ -92,10 +91,12 @@ const ProjectForm: FC<Props> = ({
             <SettingsGridCard>
                 <div>
                     {warehouse && getWarehouseIcon(warehouse)}
-                    <LeftPanelTitle>
-                        <H5>Warehouse connection</H5>
-                        <DocumentationHelpButton href="https://docs.lightdash.com/get-started/setup-lightdash/connect-project#warehouse-connection" />
-                    </LeftPanelTitle>
+                    <Flex align="center">
+                        <Title order={5} style={{ marginRight: '0px' }}>
+                            Warehouse connection
+                        </Title>
+                        <DocumentationHelpButton />
```

Comment:
looking great! one last thing, let's change keep the href props here and add back the href prop in `DocumentationHelpButton` pls 🙏

Reply:
Ok just wanted to clarify that when i keep the href in DocumentationHelpButton it will still be dependent on ActionIcon defined in DocumentationHelpButton file is it fine?

Reply:
Hopefully https://github.com/lightdash/lightdash/pull/6583#discussion_r1284128674 makes sense @jayanand05 ! 

and also this: https://github.com/lightdash/lightdash/pull/6583#discussion_r1284130972